### PR TITLE
Add initial Georgian translation

### PR DESCRIPTION
**Pull request by EkaterinePapava**
- [ ] *The Jira issue number for this PR is: MDEV-_____*

## Description

Adding initial translation of messages for Georgian language

## How can this PR be tested?

TODO: modify the automated test suite to verify that the PR causes MariaDB to
behave as intended. Consult the documentation on
["Writing good test cases"](https://mariadb.org/get-involved/getting-started-for-developers/writing-good-test-cases-mariadb-server).
In many cases, this will be as simple as modifying one `.test` and one `.result`
file in the `mysql-test/` subdirectory. Without _automated_ tests, future regressions
in the expected behavior can't be automatically detected and verified.

If the changes are not amenable to automated testing, please explain why not and
carefully describe how to test manually.

## Basing the PR against the correct MariaDB version
- [x] *This is a new feature and the PR is based against the latest MariaDB development branch*
- [ ] *This is a bug fix and the PR is based against the earliest maintained branch in which the bug can be reproduced*

## Backward compatibility
TODO: fill details here, if applicable, or remove the section

## PR quality check
- [x] I checked the [CODING_STANDARDS.md](https://github.com/MariaDB/server/blob/11.0/CODING_STANDARDS.md) file and my PR conforms to this where appropriate.

### CODE REVIEW

File: sql/share/errmsg-utf8.txt [hunk 1]
```diff
@@ -1536,6 +1596,7 @@ ER_DUP_ENTRY 23000 S1009
         est "Kattuv väärtus '%-.192T' võtmele %d"
         fre "Duplicata du champ '%-.192T' pour la clef %d"
         ger "Doppelter Eintrag '%-.192T' für Schlüssel %d"
+	geo "დუბლირებული ჩანაწერი '%-.192s' გასაღებისთვის %d"
```
Comment (grooverdan) [inline sql/share/errmsg-utf8.txt:1599]: 192s -> 192T

**Review by grooverdan — COMMENTED** [review]
You may also want to consider a locale change like #2487.

I you've able to squash these up and `git push -f` that would be appreciated.

File: sql/share/errmsg-utf8.txt (hunk 1 above)
Comment (EkaterinePapava) [reply sql/share/errmsg-utf8.txt]: fixed it.

**Comment by EkaterinePapava** [comment]
> You may also want to consider a locale change like #2487.
> 
> Can you please squash these up and `git push -f` and that would be appreciated.

Could you please point out how to do it? I'm not very comfortable with git yet.

**Comment by grooverdan** [comment]
> > You may also want to consider a locale change like #2487.
> > Can you please squash these up and `git push -f` and that would be appreciated.
> 
> Could you please point out how to do it? I'm not very comfortable with git yet.

```
git rebase -i 11.0
(edit every commit beginning after the first to be `s` rather than `pick`)
git push -f
```

**Comment by grooverdan** [comment]
also add `usr/share/mysql/georgian` to debian/mariadb-server-core.install

**Comment by EkaterinePapava** [comment]
It's all becoming big mess, will recreate PR.

**Comment by ottok** [comment]
No need to create a new MR. Just git force push the final version.

**Pull request closed without merging**